src/CoreBundle/Resources/public/javascript/tableEditor.js

```diff
@@ -585,10 +585,6 @@ function SaveViaAjaxCallback(data, statusText) {
 
         $('#tableEditorContainer .navbar-brand').html(data.name);
         $('#cmsbreadcrumb .breadcrumb-item:last').html(data.name);
-
-        if (self !== top) {
```

Comment:
This only matches/targets the describe problem in the media manager?

Or ar there other "popup-like things"?

Reply:
It is not media manager related. It is related to all table editors in frames (we don't have such a testcase in the default shop).